content/en/hypervisor.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Hypervisor
+status: Feedback Appreciated
+category: Technology
+tags: ["application", "", ""]
+---
+
+## What it is
+
+The Hypervisor is a software application that performs [virtualization](/virtualization/)
+by taking the advantage of [bare metal machine](/bare-metal-machine/) resources
+(CPU, Memory, Network, and Storage) , dividing them into sub-parts, 
+and allocating resources according to business logic to create [virtual machines (VM)](/virtual-machine/) and the ability to run multiple VMs simultaneously.
+In the marketplace, they are available as Type 1 hypervisors that can be installed directly on the physical server, 
+giving it direct access to system hardware resources.
+Type 2 hypervisors, on the other hand, are installed and 
+executed by the operating system and allow the operating system to make hardware decisions.
+## Problem it addresses
+
+Traditionally, a server could only run one operating system at a time.
+Devoting all the resources of the host machine to a single service was not the most effective way.
+Considering the capabilities of a server it can run multiple applications.
```

Comment:
The content of the rest of this section is not very accurate, or easy to follow. The aspect of having wasted a whole powerful server on just one app is an important aspect that can be followed up on. For example just saying, that a server could hold more apps, but then you have isolation and noisy neighbors problems (one highly resource-consuming app has an impact on the other ones). 
Another problem of the world before virtualization is setup costs. VMs are more quickly set up than bare metal. This is one step to closing the gap between dev and ops.

Reply:
I'm thinking these points could go into this section " acquiring a physical machine takes time, it requires a team of engineers to monitor them, whereas VMs are more quickly set up than bare metal", what do you think? @iamNoah1 and is it because a hypervisor takes away the low-level complexities (like abstraction layer over the physical hardware) so that it is easier to set up a VM whether they are of the dev team or Ops team?